dee-pabst/week4/stonksflix/views/layout.erb

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Stonkflix</title>
+    <link rel="stylesheet" href="css/style.css">
```

Comment:
put a `/` in `href="css/style.css"` and it should show pink in the background for the results page